includes/wc-template-functions.php

```diff
@@ -595,10 +595,20 @@ function wc_get_terms_and_conditions_checkbox_text() {
  * Get the privacy policy text, if set.
  *
  * @since 3.4.0
+ * @param string $type Type of policy to load. Valid values include registration and checkout.
  * @return string
  */
-function wc_get_privacy_policy_text() {
-	return trim( apply_filters( 'woocommerce_get_privacy_policy_text', get_option( 'woocommerce_checkout_privacy_policy_text', __( 'Your personal data will be used to process your order, support your experience throughout this website, and for other purposes described in our [privacy_policy].', 'woocommerce' ) ) ) );
+function wc_get_privacy_policy_text( $type = '' ) {
+	switch ( $type ) {
+		case 'checkout':
+			$text = get_option( 'woocommerce_checkout_privacy_policy_text', __( 'Your personal data will be used to process your order, support your experience throughout this website, and for other purposes described in our [privacy_policy].', 'woocommerce' ) );
+			break;
+		case 'registration':
+			$text = get_option( 'woocommerce_registration_privacy_policy_text', __( 'Your personal data will be used to support your experience throughout this website, to manage access to your account, and for other purposes described in our [privacy_policy].', 'woocommerce' ) );
+			break;
+	}
```

Comment:
This needs a default or something, otherwise `wc_get_privacy_policy_text()` will return an uninitialized `$text`